plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/natures/DataflowJavaProjectNature.java

```diff
@@ -59,67 +57,32 @@ public void setProject(IProject project) {
    */
   public static void addDataflowJavaNatureToProject(IProject project, IProgressMonitor monitor)
       throws CoreException {
-    if (isProjectAccessible(project)) {
+    Preconditions.checkNotNull(project);
+    if (!project.isAccessible()) {
```

Comment:
It looks like the boolean has reversed here? On the left this happens if the project is accessible, here not accessible.